redis_broker.go

```diff
@@ -64,15 +71,15 @@ func (cb *RedisCeleryBroker) SendCeleryMessage(message *CeleryMessage) error {
 func (cb *RedisCeleryBroker) GetCeleryMessage() (*CeleryMessage, error) {
 	conn := cb.Get()
 	defer conn.Close()
-	messageJSON, err := conn.Do("BLPOP", cb.queueName, "1")
+	messageJSON, err := conn.Do("BLPOP", cb.routes[defaultRoute].Queue, "1")
```

Comment:
I was thinking that if the user wanted to _consume_ more than one queue they would instantiate additional brokers and set those default routes accordingly.  Consuming from multiple queues on a single broker (and having to specify an opinion on the order, maybe needing to include some way to adjust the order) seems to me like it is outside the scope of this PR.

A nice side benefit of consuming only on the default queue is that it keeps this PR self-consistent, since both AMQP and Redis would therefore the same behaviour.

Reply:
In this case I probably missed something. What is the point of having a list of routes if we actually can only use one? I don't grasp why we would have both `AddRoute()` and `SetDefaultRoute()`. I'm not familiar with AMQP so I may be missing something completely here, sorry if that's the case. If you get what I may be missing I'm all hears :slightly_smiling_face: 

Anyway, I do believe this can work like this and could totally be the first step for this implementation to be merged.

Reply:
The problem with automatically consuming all queues occurs when you have task workers that are spread across multiple systems and _not every worker can run every task_.  In my case I have certain tasks that can only be executed by Windows workers while others can only be executed by a certain Linux machine.  I thought this was one of the main use cases for additional and custom routes.  Check out the example described here: https://docs.celeryproject.org/en/latest/userguide/routing.html under Basics -> Automatic Routing.

If every queue defined in every route is automatically consumed by the local broker then the above example will fail when your local worker is pulling tasks that it is not able to execute from a queue that it is not supposed to be consuming.  There must be a way to push messages to a queue that is not consumed.

Does that make sense?  Am I missing something?

Reply:
I totally understand, it does show that we have different needs and do think of the feature with our own biases :smile:. Should be fine for me anyway. I personally think it is fine like this, sligthly inefficient in my case of multiple queues on the same worker, but the overhead is some memory, an additional network connection to the redis broker and so barely anything, should be perfectly fine for my use case, and makes sense more generally.
So I'm fine with the current status.